src/components/views/Booktrak/BooktrakUtils.tsx

```diff
@@ -0,0 +1,49 @@
+import { ModelsBookListing } from "wso-api-client/lib/services/types";
+
+export const BookConditionEnumToString = (
+  condition: ModelsBookListing.ConditionEnum
+): string => {
+  const ListingConditionEnum = ModelsBookListing.ConditionEnum;
+  switch (condition) {
+    case ListingConditionEnum.Empty:
+      return "";
+    case ListingConditionEnum.POOR:
+      return "Poor";
+    case ListingConditionEnum.FAIR:
+      return "Fair";
+    case ListingConditionEnum.GOOD:
+      return "Good";
+    case ListingConditionEnum.VERYGOOD:
+      return "Very Good";
+    case ListingConditionEnum.LIKENEW:
+      return "Like New";
+    case ListingConditionEnum.NEW:
+      return "New";
+    default:
+      return "";
+  }
+};
+
+export const BookConditionStringToEnum = (
+  condition: string
+): ModelsBookListing.ConditionEnum => {
+  const ListingConditionEnum = ModelsBookListing.ConditionEnum;
+  switch (condition) {
+    case "":
```

Comment:
nit: Move this case to the bottom and remove its return since it's identical to default case.